### PR TITLE
Banners

### PR DESCRIPTION
**Pull request by dzucconi**

### CODE REVIEW

**Comment by dzucconi** [comment]
Will have to deploy API before this is merged and handle a dismissal migration.

**Comment by dzucconi** [comment]
I have to also wire up a few buttons I forgot about.

File: react/components/Banners/index.js [hunk 1]
```diff
@@ -22,29 +27,51 @@ const ButtonLink = styled(GenericButtonLink).attrs({
...
 
-export const Confirm = props => (
-  <Banner bg="state.neutral" color="gray.base" iconName="Clipboard" isCloseable={false} {...props}>
-    <Box>
-      Please check your email to confirm your account.
-      {' '}
-      Haven’t received an email yet?
-    </Box>
+const __Confirm__ = ({ resendConfirmationEmail, ...rest }) => {
+  const [mode, setMode] = useState('resting');
```
Comment (dzucconi) [inline react/components/Banners/index.js]: ↩️

Comment (broskoski) [reply react/components/Banners/index.js]: Wowowoww

File: react/components/GlobalNavElements/index.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+import TopBar from 'react/components/TopBar';
+import FixedWrapper from 'react/components/UI/FixedWrapper';
+import BottomBanner from 'react/components/BottomBanner';
+
+import WithSerializedMe from 'react/hocs/WithSerializedMe';
+
+import globalNavElementsQuery from 'react/components/GlobalNavElements/queries/globalNavElements';
+
+// Mixin serializedMe, merge it with me, pass it down through TopBar
```
Comment (broskoski) [inline react/components/GlobalNavElements/index.js]: 👍

File: react/components/UI/FixedWrapper/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+import Box from 'react/components/UI/Box';
+
+const FixedWrapper = styled(Box)`
```
Comment (broskoski) [inline react/components/UI/FixedWrapper/index.js]: Thank you

**Comment by dzucconi** [comment]
API is up. Merging this for staging; plz do give this a once over @broskoski

**Pull request merged**